api/editor/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM maven:3.6.3-openjdk-17 AS base
 WORKDIR /project
 COPY . .
 WORKDIR /project/api-editor
+
+RUN mvn clean
```

Comment:
Trying to ensure builds don't use any cached packages.  Due to the fact we're using SNAPSHOTS with the same version number most builds it can drive you crazy when you wait for a new build and it doesn't have the latest version of the package.